pkg/marathon/client_test.go

```diff
@@ -1,9 +1,17 @@
 package marathon
 
 import (
+	"fmt"
+	"io"
 	"net/http"
+	"strings"
 	"testing"
 
+	"github.com/dcos/dcos-core-cli/pkg/mocks"
+	marathon "github.com/gambol99/go-marathon"
```

Comment:
We should come up with a name that we use everywhere when importing `github.com/gambol99/go-marathon` as having it as `marathon` here while being in a package called `marathon` makes the situation confusing. I'm thinking about "goMarathon`, what do you think? cc @janisz